dlp/dlp_inspect_image_file.rb

```diff
@@ -0,0 +1,60 @@
+# Copyright 2018 Google, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# [START dlp_inspect_image_file]
+require "google/cloud/dlp"
+
+def inspect_image_file (
+    project_id = "YOUR_PROJECT_ID",
+    filepath   = "path/to/image.png"
+  )
+
+  # Instantiate a client
+  dlp = Google::Cloud::Dlp.new
+
+  # Get the bytes of the file
+  file_bytes = File.open(filepath, "rb").read()
+
+  # Construct request
+  parent         = "projects/#{project_id}"
+  item           = { byte_item: { type: :IMAGE_PNG, data: file_bytes } }
+  inspect_config = {
+    # The infoTypes of information to match
+    info_types: [
+      {name: 'PHONE_NUMBER'},
+      {name: 'EMAIL_ADDRESS'},
+      {name: 'CREDIT_CARD_NUMBER'},
```

Comment:
Two style elements:
* Include a space inside the braces (https://docs.rubocop.org/en/latest/cops_layout/#layoutspaceinsidehashliteralbraces)
* Be consistent with quoting style (single vs double). The current preference is to prefer double quotes. (https://docs.rubocop.org/en/latest/cops_style/#stylestringliterals)

better: `{ name: "PHONE_NUMBER" }`